src/views/order_entry_view.js

```diff
@@ -0,0 +1,62 @@
+import Backbone from 'backbone';
+import _ from 'underscore';
+
+import Quote from '../models/quote';
+import Order from '../models/order';
+
+import OrderView from '../views/order_view';
+import QuoteView from '../views/quote_view';
+
+const OrderEntryView = Backbone.View.extend({
+  initialize(params){
+    this.template = params.template;
+    this.bus = params.bus;
+    this.quotesList = params.quotesList;
+  },
+
+  events: {
+    'click button.btn-buy': 'buy',
+    'click button.btn-sell': 'sell'
+  },
+  buy(event) {
+    event.preventDefault();
+    console.log('test1');
+    let orderData = {
```

Comment:
If you take out the `console.log`s, this function and `sell()` are almost identical. Could you consolidate the two somehow?